cves/kernel/CVE-2022-3108.yml

```diff
@@ -448,7 +452,7 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: I don't think it easily falls under slip, lapse or planning as it is a minor coding mistake. It's really just a forgotten or unplanned if statement, not checking if a function returns null.
```

Comment:
If you look at the example of lapse it has forgetting to check null.

Reply:
Oh yeah